src/lib/form-field/_form-field-standard-theme.scss

```diff
@@ -20,6 +22,13 @@
     &.mat-form-field-disabled .mat-form-field-underline {
       @include mat-control-disabled-underline($underline-color);
     }
+
+    &:not(.mat-form-field-invalid):not(.mat-focused) {
```

Comment:
It looks like the same block is repeated here and in the outline theme. Otherwise I can see us forgetting to update one of the two if we ever decide to change it.

Reply:
The outline and standard themes look a bit different. I think perhaps you meant the standard and fill? I have created a mixin for those two appearances.